api/graphql/opening_hours/opening_hours_types.py

```diff
@@ -24,17 +27,19 @@ def resolve_start_time(self, info):
         if not self.start_time:
             return None
         tzinfo = self.start_time.tzinfo or DEFAULT_TIMEZONE
-        start = tzinfo.localize(self.start_time)
-
-        return start
+        start_dt = datetime.datetime.combine(datetime.date.today(), self.start_time)
```

Comment:
I am having a hard time understanding what these three lines do. I understand they do timezone conversion, but for quickly figuring out what happens, some commenting would help.

It also seems that these three lines are duplicated in many places. If these lines would be extracted into a helper function, this would aid in understanding code, and further cases.

Reply:
For some reason this was in "another review" so had to post multiple reviews :thinking:

Reply:
yes, this was tricky and took me some time to fiddle with this part of the code since this will actually change the gql api to return always UTC opening hours since for some reason the timezone information was not rendered to the time value when it was set to `Europe/Helsinki` but only when it was pytz.UTC. Also why converting the time to datetime was that the localize method does not work for new timezone :exploding_head: 

Util function would be a good thing to be used here :+1:  I'll do the changes.